.travis.yml

```diff
@@ -0,0 +1,121 @@
+sudo: required
+dist: precise
```

Comment:
there's no requirements, so i just took something that works (based on logrotate project setup)